class2.html

```diff
@@ -245,9 +245,9 @@ <h2>Reminders</h2>
 
         <section>
           <h2>Block vs Inline HTML Elements</h2>
-          <img src="images/block-vs-inline-element.jpeg"
+          <img src="images/block-vs-inline-element.png"
```

Comment:
I'm curious to hear your thoughts on why you prefer the new image vs the current image. I personally don't find the new image as clear when illustrating the different behavior of an inline element vs a block element but I'd love to hear your thoughts!

Reply:
I think you're right, and the original image is better. I ended up here for two reasons. First, this image matches the other images from interneting is hard, and I like that. Second, I didn't really click with the other image for reasons I'm not totally sure of. Something about it is _true_ but doesn't feel very explanatory.

Maybe it's the color blocks don't feel like they have a sense of scale, so it feels more like literal blocks of color, not elements on the page. Maybe my mental model just doesn't sync up to the image but everyone else gets it. Not sure.

Either way, I'll revert it for now and look for a better image later.

Reply:
@edwardloveall FWIW, since you're teaching class 2, I'm happy to use a different image, one that resonates with you and facilitates how you'd like to present the information. Don't let my personal "it doesn't resonate as well with me" prevent you from using an image that does resonate with you. :)

Reply:
Thanks, but I had a similar reaction in the back of my head that the image I picked wasn't quite right and I ignored it. Your comment rang true.

We could try the one I picked, but I'd rather have the old one bug me so I'll find a better one. 😄